app/src/main/res/layout/activity_collection.xml

```diff
@@ -21,13 +21,7 @@
             android:id="@+id/secondTab"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:text="@string/insects" />
-
-        <android.support.design.widget.TabItem
-            android:id="@+id/thirdTab"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:text="@string/underwater" />
```

Comment:
I guess you removed this so that it doesn't show ACNL data with the ACNH data?

Reply:
Yup, we will put the ACNL data back once we have a way to switch games